Dockerfile

```diff
@@ -45,3 +41,12 @@ RUN mkdir /root/.ssh/
 RUN touch /root/.ssh/environment
 
 CMD env >> /root/.ssh/environment; export -p | grep _ >> /etc/profile; /usr/sbin/sshd -D;
+
+# settings default django settings module (2nd part)
+ENV POSTGRES_PORT_5432_TCP_PORT="5432"
+ENV POSTGRES_PORT_5432_TCP_ADDR="172.17.0.2"
```

Comment:
I will investigate if it needed but me and @WojtekReu tested that it's needed and then works... (but with less iterations... not for each env var).